aries-site/src/data/mockData/firmware-bundles.json

```diff
@@ -53,7 +53,7 @@
         "selfUri": "/v1/firmware-bundles/1949ad1a5cc488fbdbe751863b6b8376",
         "name": "Service Pack for ProLiant",
         "displayName": "2020.09.1(22 Apr 2021)",
-        "description": "The Service Pack for ProLiant (SPP) is a comprehensive systems software and firmware update solution, which is delivered as a single ISO image. This Solution uses Smart Update Manager (SUM) as the deployment tool and is tested on all HPE ProLiant Gen9 and Gen10 servers as defined in the Service Pack for ProLiant Server Support Guide found at www.hpe.com/servers/spp/documentation.",
+        "description": "The Service Pack for ProLiant (SPP) is a comprehensive systems software and firmware update solution, which is delivered as a single ISO image.",
```

Comment:
I don't think this data should change. If we are worried about length, then the component should truncate the content, not delete the data.

Reply:
okay Illl truncate